metricbeat/module/ceph/_meta/docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+== ceph Module
+
+This is the ceph Module.
+
+We used code/reference from:
+https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go
```

Comment:
Actually I copy/paste code and then adapted to collect metrics. It's really similar but not exactly equal. That's why I wrote this on docs.asciidoc. Is it a problem?